cmake/projects/FLAC/hunter.cmake

```diff
@@ -0,0 +1,39 @@
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_cmake_args)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME
+    FLAC
+    VERSION
+    1.3.3-p0
+    URL
+    "https://github.com/drodin/flac/archive/1.3.3-p0.tar.gz"
```

Comment:
This should be forked into cpp-pm org and should go through a code review

Reply:
can you fork it?

Reply:
Forked here https://github.com/cpp-pm/flac and added you to the `cpp-pm-packages` team for write access.

Be careful pushing your branch to that fork - any branch `hunter*` can be pushed once and then requires a PR for further updates.

Reply:
thanks, I've pushed hunter branch and changed the package source to cpp-pm repo
could you please also fork SFML https://github.com/SFML/SFML into cpp-pm org

Reply:
Done here: https://github.com/cpp-pm/SFML